src/Altinn.AccessManagement.Core/Helpers/RightsHelper.cs

```diff
@@ -53,16 +53,106 @@ public static List<AttributeMatch> GetDelegationSubjectAttributeMatches(Delegati
         }
 
         /// <summary>
-        /// Builds a RightsQuery request model for lookup of a users rights for a given resource registry service on behalf of the given reportee party
+        /// Builds a RightsQuery request model for lookup of a users rights for a given service resource on behalf of the given reportee party
         /// </summary>
-        public static RightsQuery GetRightsQueryForResourceRegistryService(int userId, string resourceRegistryId, int fromPartyId)
+        public static RightsQuery GetRightsQuery(int userId, int fromPartyId, string resourceRegistryId = null, string org = null, string app = null)
         {
+            if (!string.IsNullOrEmpty(org) && !string.IsNullOrEmpty(app))
+            {
+                return new RightsQuery
+                {
+                    To = new List<AttributeMatch> { new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.UserAttribute, Value = userId.ToString() } },
+                    From = new List<AttributeMatch> { new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.PartyAttribute, Value = fromPartyId.ToString() } },
+                    Resource = new List<AttributeMatch> { new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.OrgAttribute, Value = org }, new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.AppAttribute, Value = app } }
```

Comment:
Shouldn't this reflect that it can be more than one resource?
```suggestion
                    Resources = new List<AttributeMatch> { new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.OrgAttribute, Value = org }, new AttributeMatch { Id = AltinnXacmlConstants.MatchAttributeIdentifiers.AppAttribute, Value = app } }
```

Reply:
It's not multiple resources though. It's a single resource identified by a list of attributes

Reply:
What's the use case for when there are more than one item in the list?

Reply:
When you need more then one attribute to identify a resource. e.g.:
- Org and App to identify an Altinn app
- ServiceCode and ServiceEditionCode to identify an Altinn 2 service

Reply:
Oh shoot, how many possible permutations are we talking here?

Reply:
ahh nvm, we don't have to worry about that because we just receive that response with an undefined length of the list but when we delegate it we just send in the same list and the reource will be delegated.